internal/trafficcontrol/tc.go

```diff
@@ -35,6 +35,14 @@ var (
 	udpProtoStr = "udp"
 )
 
+type VFInfo struct {
+	Name          string
+	PfName        string
+	Index         int
+	VlanID        int
+	Authenticated bool
```

Comment:
Instead of tracking this state identically here, is it better to have a link to a parent structure that represents the PF?